### PR TITLE
Jetpack Connect: Manual instructions copy + illustration updates

### LINKED ISSUE #22716: Jetpack Connect: Update "Connect to WordPress.com" image and copy on "Ready for Installation" step
#### Steps to reproduce
1. Starting at URL: https://wordpress.com/jetpack/connect
2. Enter the URL of a self-hosted site without Jetpack installed and select "Connect Now."

#### What I expected

I expected the instructions on the "Ready for Installation" screen to match the flow.

#### What happened instead

The "Ready for Installation" instructions show the old "Connect to WordPress.com" button and copy:

screenshot 2018-02-22 17 40 34

Those should be updated to reflect the new "Set up Jetpack" button:

screenshot 2018-02-22 17 44 06

#### Browser / OS version

macOS 10.13.3 / Chrome 64

#### Context / Source
#bug-scrub #manual-testing

### LINKED ISSUE #22919: UI: Discrepancies in Calypso Selector Jetpack Flow
# Summary

This screen:

screen shot 2018-03-01 at 18 48 39

...could use a few small UI/text updates:

### Screenshot

a 1 it s not a link it s a button
a 2 activate plugin button not activate link

### Suggested text update:

> Then you'll click the blue "Activate" link to activate Jetpack

 → Then click the blue "Activate Plugin" button to activate Jetpack.

### Screenshot
b it s not _install_ but _install now_ inconsistent instruction vs ui

### Suggested text update:

> Click the blue install button.

 → Click the blue Install Now button.

## c) "Set up Jetpack" instead of "Connect to WordPress.com"

d this button should be called set up jetpack

d set up jetpack screen

### Suggested text update:

> Finally, click the "Connect to WordPress.com" button to finish the process.

 → Finally, click the "Set up Jetpack" button to finish the process.

#### Context / Source

#manual-testing

### PR DESCRIPTION
**Pull request by seear**
Fixes #22919, fixes #22716.

Update manual install and activate instructions to match wp-admin.

The copy and styling updates are a combination of the suggestions from #22919, #22716, and the mockups at p7rd6c-1dS-p2.

**Install Instructions Before**

install_before

**Install Instructions After**

install_after

**Activate Instructions Before**

activate_before

**Activate Instructions After**

activate_after

## Testing

**Install + Activate Instructions**
* Go to https://calypso.live/jetpack/connect?branch=update/jetpack-connect/install-instructions
* Enter the URL of a site without the Jetpack plugin installed

**Activate Instructions**
* Go to https://calypso.live/jetpack/connect?branch=update/jetpack-connect/install-instructions
* Enter the URL of a site with the Jetpack plugin installed but not activated

### CODE REVIEW

File: client/jetpack-connect/example-components/jetpack-connect.jsx [hunk 1]
```diff
@@ -45,14 +45,14 @@ const JetpackConnectExampleConnect = ( { isLegacy, url, translate, onClick } ) =
...
-								{ translate( 'Your Jetpack is almost ready!', {
+								{ translate( 'Connect Jetpack to WordPress.com', {
 									context:
 										'Jetpack Connect activate plugin instructions, connection banner headline',
 								} ) }
 							</div>
 						) : null }
 						<div className="example-components__content-wp-admin-connect-button" aria-hidden="true">
-							{ translate( 'Connect to WordPress.com', {
+							{ translate( 'Set up Jetpack', {
```
Comment (a8ci18n) [inline client/jetpack-connect/example-components/jetpack-connect.jsx]: :information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?

File: client/jetpack-connect/example-components/jetpack-connect.jsx [hunk 2]
```diff
@@ -45,14 +45,14 @@ const JetpackConnectExampleConnect = ( { isLegacy, url, translate, onClick } ) =
 					<div className="example-components__content-wp-admin-connect-banner">
 						{ ! isLegacy ? (
 							<div className="example-components__content-wp-admin-plugin-name" aria-hidden="true">
-								{ translate( 'Your Jetpack is almost ready!', {
+								{ translate( 'Connect Jetpack to WordPress.com', {
```
Comment (a8ci18n) [inline client/jetpack-connect/example-components/jetpack-connect.jsx]: :information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?

File: client/jetpack-connect/example-components/jetpack-connect.jsx [hunk 3]
```diff
@@ -45,17 +45,11 @@ const JetpackConnectExampleConnect = ( { isLegacy, url, translate, onClick } ) =
 					<div className="example-components__content-wp-admin-connect-banner">
 						{ ! isLegacy ? (
 							<div className="example-components__content-wp-admin-plugin-name" aria-hidden="true">
-								{ translate( 'Your Jetpack is almost ready!', {
-									context:
-										'Jetpack Connect activate plugin instructions, connection banner headline',
-								} ) }
+								{ translate( 'Connect Jetpack to WordPress.com' ) }
```
Comment (a8ci18n) [inline client/jetpack-connect/example-components/jetpack-connect.jsx]: Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'Connect Jetpack to WordPress.com.' )` *ES Score: 12*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

**Comment by joanrho** [comment]
Pushed some commits to edit the copy. 

Also, I saw the spinner parts were a bit misaligned. Do you see that on your end too? If so, is that fixable? Might be out of scope for this particular PR. 

screen shot 2018-03-08 at 12 31 52 pm


As for text + illustrations, LGTM! Thanks @seear !

**Comment by seear** [comment]
> I saw the spinner parts were a bit misaligned

We don't do anything custom with the spinner for Jetpack Connect, so it should match up with the [new design](https://github.com/Automattic/wp-calypso/pull/23119#pullrequestreview-102330257).

Thanks for the review and updates @joanrho !

**Pull request merged**